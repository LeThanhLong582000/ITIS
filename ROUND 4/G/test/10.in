8
109
17 78 38 99 72 64 17 87 63 57 105 72 80 75 61 32 36 86 86 16 55 48 70 37 1 32 68 81 30 43 62 1 36 37 72 52 79 18 26 49 9 65 34 55 47 19 63 7 87 88 45 109 33 107 76 7 76 36 13 46 45 94 47 85 66 24 107 47 107 64 39 24 25 83 19 94 99 42 99 26 4 105 15 49 87 90 106 94 40 101 107 18 94 44 83 17 3 70 26 25 22 103 22 56 30 27 103 15 21 
378
76 130 348 98 185 178 372 209 156 241 287 278 263 154 299 304 307 113 263 29 291 303 168 289 169 282 36 270 105 219 323 84 337 39 172 192 122 205 123 38 60 350 36 153 211 246 302 266 223 205 78 223 143 264 287 311 324 327 100 50 54 175 319 345 230 161 270 292 336 321 54 179 60 198 112 273 304 219 268 12 57 115 43 222 301 139 217 84 85 183 327 153 94 249 20 342 65 350 89 368 233 270 87 93 343 367 284 112 8 193 304 20 97 274 134 286 250 38 290 114 228 270 66 140 182 371 151 69 340 288 208 276 18 3 187 241 59 344 74 104 224 32 344 163 138 373 276 128 334 40 189 100 177 4 101 150 184 75 309 324 221 320 103 23 139 252 141 124 359 95 193 105 346 320 302 286 202 128 167 18 211 323 51 56 79 340 235 166 207 168 270 320 36 316 24 90 277 160 142 49 108 262 351 330 170 251 191 146 350 71 51 95 124 67 54 127 279 17 162 353 333 25 268 28 117 165 185 94 149 153 211 361 11 76 22 330 369 27 30 3 127 132 263 346 207 176 142 187 38 149 113 78 332 184 2 41 254 113 32 51 309 344 347 356 62 16 352 215 75 311 1 236 195 167 137 18 128 200 70 331 264 59 372 281 195 165 47 200 147 184 128 123 212 181 262 44 146 130 172 260 47 106 275 169 23 14 187 269 2 271 296 59 61 169 4 50 269 314 3 339 170 165 265 86 237 117 266 82 66 13 314 316 218 203 368 10 350 27 46 236 238 277 148 288 77 226 248 262 96 346 294 331 272 161 176 304 282 340 299 301 346 149 264 155 264 88 260 287 1 262 276 16 209 98 84 309 62 279 
338
181 86 148 83 296 109 171 97 131 279 209 57 81 299 252 199 177 108 15 312 290 308 78 313 330 30 6 71 314 90 52 75 307 11 171 37 228 310 195 74 18 205 293 148 9 148 278 205 237 109 183 104 119 308 31 328 100 54 1 53 165 202 334 54 166 179 201 253 224 94 338 223 39 284 305 104 79 44 252 330 151 61 276 268 244 91 167 210 173 181 148 325 135 129 227 163 191 278 58 263 64 118 222 35 113 156 235 119 302 311 103 175 108 65 175 287 195 171 326 184 249 338 56 199 28 245 331 203 247 213 89 91 91 236 197 226 37 323 44 103 236 111 184 126 28 269 206 282 315 171 235 117 18 254 160 306 97 121 111 274 218 46 128 45 87 194 288 143 197 2 164 114 324 106 24 288 224 197 33 119 189 310 220 220 331 180 63 313 56 31 29 22 235 10 207 200 273 202 194 264 71 164 11 244 117 75 41 189 322 193 262 128 91 136 234 41 228 289 332 189 226 238 45 188 307 176 130 333 57 83 4 75 95 32 138 239 286 224 231 266 326 275 321 75 129 311 21 298 28 336 90 92 190 63 67 101 141 219 276 223 198 205 185 257 215 99 270 58 161 183 305 46 153 267 115 137 38 269 68 56 33 80 328 327 175 306 202 37 336 188 103 104 22 18 182 272 260 39 96 334 289 295 321 297 136 263 203 112 283 136 129 267 311 201 16 70 250 94 116 163 148 130 177 110 151 77 10 283 293 263 255 286 186 134 66 306 1 317 
355
128 68 140 265 239 302 338 44 134 279 171 279 143 50 244 197 125 44 35 348 257 256 337 211 12 86 292 196 36 69 100 11 159 314 348 63 264 129 325 111 19 206 152 209 96 322 249 98 70 324 352 117 303 6 198 321 34 169 302 76 183 82 202 1 147 199 112 212 163 344 93 152 143 40 189 283 248 301 30 197 104 208 185 252 301 262 341 163 192 235 173 38 343 258 271 24 275 327 148 37 141 224 291 232 159 74 243 75 260 220 204 85 337 127 258 38 110 230 120 296 337 25 38 136 125 204 62 295 313 149 321 87 256 338 291 174 209 246 219 128 221 194 213 136 326 140 160 149 205 332 239 318 30 317 106 277 107 30 138 271 117 63 148 261 168 253 141 121 72 272 323 299 99 221 105 111 219 181 186 127 128 110 94 246 152 92 206 126 132 343 38 51 172 19 92 133 49 58 115 13 312 136 117 187 191 342 10 58 343 47 203 160 187 91 258 222 347 225 100 152 313 70 113 95 281 342 138 284 132 347 204 67 3 99 239 241 242 312 202 285 168 25 108 30 92 341 22 196 313 113 209 309 24 183 257 262 19 282 245 231 66 51 258 345 32 251 234 232 3 82 41 218 67 268 205 292 72 189 308 167 39 202 6 147 153 251 185 133 200 48 138 188 266 295 258 353 25 262 350 3 182 355 76 58 79 342 310 213 249 163 210 306 116 287 40 13 322 87 111 266 174 58 19 265 120 119 291 229 45 224 150 118 333 352 154 85 249 243 276 206 333 246 170 115 332 325 90 28 165 274 285 218 272 345 340 
484
189 336 450 157 431 446 182 23 168 31 385 59 294 458 299 250 236 96 403 79 150 114 224 13 364 375 427 27 13 62 350 154 268 349 390 307 144 341 4 23 269 426 159 257 285 234 362 479 123 125 223 14 462 206 268 47 148 284 438 386 336 26 83 484 397 479 304 484 189 51 388 29 213 50 56 411 71 5 242 130 96 67 256 227 360 246 188 201 202 304 222 210 239 219 96 216 292 346 54 92 266 442 126 274 80 406 468 177 124 107 459 251 480 292 22 294 98 57 348 474 42 71 161 317 315 5 298 251 244 249 173 45 307 446 24 421 284 61 181 250 287 286 177 270 265 68 73 181 189 134 356 355 104 170 422 152 130 161 352 299 91 444 316 426 263 120 275 264 265 174 320 6 407 290 465 355 394 122 239 30 280 53 257 284 71 217 126 262 379 339 141 363 169 471 215 289 152 44 427 391 201 185 311 177 83 375 136 193 25 43 119 452 292 254 9 163 394 227 58 108 70 279 305 92 463 448 311 299 249 25 165 285 201 65 366 225 397 16 316 165 115 411 258 79 158 249 423 456 53 343 65 32 259 382 378 335 346 139 412 337 474 37 29 426 417 103 260 189 115 470 72 111 102 236 475 280 265 321 291 382 369 433 67 284 404 91 249 384 182 454 83 34 438 368 436 87 117 460 22 204 11 201 319 464 140 150 128 23 437 71 261 134 274 55 121 290 145 362 21 388 299 8 346 287 260 180 283 282 402 250 223 230 215 69 442 136 27 13 251 112 189 425 400 51 398 366 341 380 65 412 75 107 115 234 422 113 176 446 13 341 117 185 442 257 170 81 46 71 252 54 270 301 450 476 18 351 416 363 60 145 353 12 361 80 349 55 349 249 398 114 60 263 282 402 137 294 139 361 374 391 191 200 450 215 441 35 421 112 372 169 17 462 457 462 285 437 483 190 426 34 291 39 251 240 27 275 383 56 228 398 455 311 247 21 148 464 105 358 7 160 426 346 254 3 213 205 269 367 93 119 461 272 358 353 353 473 307 95 392 335 93 424 227 150 173 297 445 37 413 362 469 168 262 155 104 78 250 165 280 99 368 111 362 305 
487
216 433 290 103 68 196 393 59 278 416 86 147 363 481 92 130 186 163 8 99 27 370 392 301 428 199 445 287 220 69 465 98 7 164 209 285 66 3 48 256 264 181 114 367 361 48 82 80 435 51 331 417 144 241 154 434 435 268 223 18 98 150 251 235 208 427 434 300 11 261 369 410 155 82 7 405 331 258 181 415 293 244 1 376 439 243 191 456 327 124 463 446 18 76 466 226 57 280 281 41 454 252 487 389 176 422 90 141 74 352 285 57 438 311 310 98 288 25 111 435 277 481 144 371 310 12 109 180 237 180 240 479 272 434 209 83 83 426 50 141 428 219 220 245 153 441 351 49 163 204 156 474 300 358 302 6 394 416 186 114 331 254 234 105 22 300 359 167 482 108 403 209 143 4 2 230 31 257 42 208 98 215 114 277 290 87 432 156 193 431 297 467 380 149 30 78 223 449 42 397 12 107 391 231 440 283 165 177 132 364 151 469 80 436 432 139 3 433 238 231 296 360 479 183 291 82 378 477 433 435 301 80 437 181 470 297 443 408 464 481 39 190 360 302 449 475 1 240 27 41 91 237 233 16 56 41 252 78 376 377 409 372 114 103 189 367 433 27 251 164 33 122 320 374 35 146 48 394 194 470 417 186 314 444 15 65 432 46 56 56 69 46 432 259 333 165 433 487 175 22 381 67 98 397 108 143 108 452 437 206 96 212 168 456 177 406 162 199 119 342 370 352 416 405 28 186 156 57 479 106 384 127 225 393 269 20 436 244 263 250 197 57 176 346 108 430 300 337 433 466 145 395 332 125 211 21 49 163 105 255 388 71 57 158 244 425 283 74 143 282 391 469 324 180 179 204 24 149 410 464 48 113 193 334 106 88 380 45 34 257 204 335 147 16 372 445 358 157 455 199 77 340 416 31 327 79 312 323 293 304 17 2 389 24 256 459 30 200 117 258 336 343 271 114 300 54 176 175 437 338 329 364 475 312 63 240 306 427 214 248 471 398 145 291 168 418 356 127 481 115 324 182 301 392 379 453 336 424 14 332 216 190 487 293 156 54 13 352 354 246 442 327 331 229 217 14 98 231 196 281 482 224 269 210 166 309 200 
482
47 62 77 94 457 251 212 110 38 317 349 439 217 182 255 375 186 232 447 320 275 397 394 16 15 100 119 331 176 316 211 327 266 153 309 316 383 148 62 420 429 289 114 150 421 200 46 283 338 295 179 27 213 159 147 439 150 48 104 24 313 463 262 117 223 155 391 86 347 120 340 402 203 330 220 7 84 56 86 476 14 70 268 455 104 212 311 16 463 173 109 207 451 379 100 482 184 143 47 103 33 318 87 348 352 396 61 24 86 40 186 348 63 50 431 76 49 479 360 78 123 383 240 222 127 291 406 293 433 8 396 319 312 46 119 225 205 68 248 87 333 22 400 77 60 113 229 116 3 179 29 465 428 371 124 86 103 174 93 448 472 481 277 401 236 449 412 426 247 411 385 300 103 363 208 255 374 299 154 425 28 417 324 52 34 107 110 419 203 313 206 24 110 158 118 30 399 346 363 9 173 175 153 431 323 316 437 17 455 119 199 17 305 482 291 482 48 331 481 296 5 195 381 127 243 425 263 411 482 76 159 444 343 192 269 283 152 411 216 288 56 389 433 279 442 291 279 306 396 313 446 197 471 199 187 145 428 232 454 55 320 390 365 179 184 38 288 472 155 209 398 389 455 479 451 182 250 402 261 123 419 392 10 328 42 300 113 173 82 464 269 75 261 53 184 315 22 245 9 19 188 366 102 234 56 232 482 92 49 135 18 440 195 1 343 140 14 303 274 219 217 107 363 480 158 167 395 27 258 300 123 222 435 413 480 7 351 481 19 164 396 293 383 8 18 289 262 91 308 69 62 233 183 103 430 453 463 12 328 59 36 30 139 315 157 237 4 363 20 313 395 465 390 184 473 68 477 155 277 257 152 126 396 225 368 434 390 88 98 43 369 222 240 334 437 70 478 321 73 249 445 114 94 268 156 7 208 201 114 36 440 443 162 317 18 161 142 445 184 348 285 321 42 71 267 320 121 96 184 301 242 360 85 59 182 230 399 254 344 410 334 282 132 46 190 236 467 399 467 268 370 297 9 310 142 400 164 106 226 327 19 311 377 27 12 18 329 422 254 273 358 208 253 442 29 326 66 481 57 265 380 458 
348
256 102 56 329 235 160 272 191 50 25 17 97 68 61 10 52 158 215 43 144 311 231 263 11 38 18 123 245 5 328 84 333 30 241 122 51 237 151 39 289 236 193 202 288 227 133 234 126 290 179 139 66 60 294 103 303 204 295 159 324 15 16 209 256 83 227 145 333 254 103 282 104 268 335 13 70 103 47 206 253 249 313 283 166 309 222 204 116 186 171 95 9 84 269 257 28 131 259 174 71 242 177 210 121 331 66 120 130 153 158 20 103 208 142 252 51 266 90 30 237 102 191 1 268 22 152 163 141 18 64 190 56 240 236 276 132 240 202 10 330 81 98 312 196 157 83 109 178 217 337 91 58 83 186 339 172 36 194 114 189 180 131 176 8 185 99 60 130 118 91 71 291 178 17 344 65 89 93 91 281 194 275 109 33 199 247 257 223 33 30 55 313 57 157 280 188 96 90 118 281 92 277 76 181 132 90 342 86 193 61 93 279 13 173 220 160 104 81 120 77 227 281 10 285 154 29 127 319 188 106 63 215 55 37 190 87 253 87 6 265 211 199 8 32 215 234 338 255 135 312 34 11 238 310 46 97 195 121 14 108 38 196 33 280 158 247 31 337 34 205 306 254 101 83 220 180 204 286 252 141 307 320 264 343 201 73 307 334 173 260 340 166 230 190 148 320 256 221 160 281 46 123 162 128 259 74 166 107 32 142 55 155 283 189 300 119 288 168 71 257 211 297 245 69 16 329 228 152 82 180 33 63 169 63 221 70 87 208 297 177 15 70 37 298 136 271 86 116 
