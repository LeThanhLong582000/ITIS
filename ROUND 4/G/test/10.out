53
185
174
182
247
240
239
168
